cmd/reaper/main.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/client"
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/reaper"
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/reaperconfig"
+)
+
+func main() {
+	deleteCmd := flag.NewFlagSet("delete", flag.ExitOnError)
```

Comment:
Add functionality so that if UUID flag isn't set when deleting, then user will be prompted for UUID, similar to how a reaper is configured in `create`